libs/features/icons/icons.js

```diff
@@ -46,6 +46,7 @@ function decorateToolTip(icon) {
   // Position is the next to last part of a tooltip
   const place = conf.pop()?.trim().toLowerCase() || 'right';
   icon.className = `icon icon-info milo-tooltip ${place}`;
+  icon.setAttribute('tabindex', '0');
```

Comment:
is `icon` always clickable?

Reply:
This is under decorateToolTip() which means my code affects only for tooltip.